### PR TITLE
Adds configuration for Github flows

### PR DESCRIPTION
**Pull request by AlbertoElias**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+### How to test?
+
+<!-- What steps in order should someone run to test -->
+
+## Checklist
+
+These are the criteria that every PR should meet, please check them off as you
+review them:
+
+- [ ] Include tests
```
Comment (elias-garcia) [inline .github/PULL_REQUEST_TEMPLATE.md:19]: I would remove this bullet

File: .github/PULL_REQUEST_TEMPLATE/develop.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+### How to test?
+
+<!-- What steps in order should someone run to test -->
+
+## Checklist
+
+These are the criteria that every PR should meet, please check them off as you
+review them:
+
+- [ ] Include tests
```
Comment (elias-garcia) [inline .github/PULL_REQUEST_TEMPLATE/develop.md:19]: I would remove this bullet

File: .github/PULL_REQUEST_TEMPLATE/main.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+### List of PRs
+
+<!-- List of PRs made to develop included in this PR -->
+
+## Checklist
+
+These are the criteria that every PR should meet, please check them off as you
+review them:
+
+- [ ] Update tests
```
Comment (elias-garcia) [inline .github/PULL_REQUEST_TEMPLATE/main.md:10]: I would remove this bullet

**Comment by AlbertoElias** [comment]
@elias-garcia done!

**Review by mmilenkovic — APPROVED** [review]

**Review by elias-garcia — APPROVED** [review]

**Pull request merged**